counting words in file.py

```diff
@@ -1,12 +1,45 @@
-file=open("proxy-clean-text.txt","r+")
-wordcount={}
-for word in file.read().split():
+
+
```

Comment:
We can have a comment here specifying what this method does, what is the input and the output